react/features/av-moderation/middleware.ts

```diff
@@ -208,24 +209,50 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
  */
 StateListenerRegistry.register(
     state => state['features/base/conference'].conference,
-    (conference, { dispatch }, previousConference) => {
+    (conference, { dispatch, getState }, previousConference) => {
         if (conference && !previousConference) {
             // local participant is allowed to unmute
             conference.on(JitsiConferenceEvents.AV_MODERATION_APPROVED, ({ mediaType }: { mediaType: MediaType; }) => {
                 dispatch(localParticipantApproved(mediaType));
 
-                // Audio & video moderation are both enabled at the same time.
-                // Avoid displaying 2 different notifications.
-                if (mediaType === MEDIA_TYPE.AUDIO) {
-                    dispatch(showNotification({
-                        titleKey: 'notify.hostAskedUnmute',
-                        sticky: true,
-                        customActionNameKey: [ 'notify.unmute' ],
-                        customActionHandler: [ () => dispatch(muteLocal(false, MEDIA_TYPE.AUDIO)) ],
-                        uid: ASKED_TO_UNMUTE_NOTIFICATION_ID
-                    }, NOTIFICATION_TIMEOUT_TYPE.MEDIUM));
-                    dispatch(playSound(ASKED_TO_UNMUTE_SOUND_ID));
+                // hide any existing notification as we will show a new one
+                // the case where first audio is approved and then video is approved, we avoid double notification
+                dispatch(hideNotification(ASKED_TO_UNMUTE_NOTIFICATION_ID));
```

Comment:
No need to hide it if you are just about to push a new one. If it has the same ID it will replace it.